operator/hack/addons.yaml

```diff
@@ -2,28 +2,123 @@
 # within a cluster. logcli pod helps write queries, promtail writes logs, etc
 ---
 apiVersion: v1
-kind: Pod
+kind: ServiceAccount
+metadata:
+  name: logcli-loki
```

Comment:
Can we adapt the names to use a common prefix like `lokistack-dev-addons-`, e.g. `lokistack-dev-addons-logcli`